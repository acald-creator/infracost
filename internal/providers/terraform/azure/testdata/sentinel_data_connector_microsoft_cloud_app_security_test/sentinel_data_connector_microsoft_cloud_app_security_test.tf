provider "azurerm" {
  skip_provider_registration = true
  features {}
}

resource "azurerm_resource_group" "example" {
  name     = "example-resources"
  location = "eastus"
}

# Add example resources for SentinelDataConnectorMicrosoftCloudAppSecurity below

resource "azurerm_log_analytics_workspace" "sentinel_data_connector_microsoft_cloud_app_security" {
  name                = "example-workspace"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  sku                 = "PerGB2018"
}

resource "azurerm_log_analytics_solution" "sentinel_data_connector_microsoft_cloud_app_security" {
  solution_name         = "SecurityInsights"
  location              = azurerm_resource_group.example.location
  resource_group_name   = azurerm_resource_group.example.name
  workspace_resource_id = azurerm_log_analytics_workspace.sentinel_data_connector_microsoft_cloud_app_security.id
  workspace_name        = azurerm_log_analytics_workspace.sentinel_data_connector_microsoft_cloud_app_security.name

  plan {
    publisher = "Microsoft"
    product   = "OMSGallery/SecurityInsights"
  }
}

resource "azurerm_sentinel_data_connector_microsoft_cloud_app_security" "sentinel_data_connector_microsoft_cloud_app_security" {
  name                       = "example"
  log_analytics_workspace_id = azurerm_log_analytics_solution.sentinel_data_connector_microsoft_cloud_app_security.workspace_resource_id
}
