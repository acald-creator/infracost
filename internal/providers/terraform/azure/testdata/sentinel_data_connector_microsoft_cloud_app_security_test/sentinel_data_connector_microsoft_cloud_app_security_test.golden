
 Name                                                                                    Monthly Qty  Unit            Monthly Cost 
                                                                                                                                   
 azurerm_log_analytics_workspace.sentinel_data_connector_microsoft_cloud_app_security                                              
 ├─ Log data ingestion                                                                 Monthly cost depends on usage: $2.99 per GB 
 └─ Log data export                                                                    Monthly cost depends on usage: $0.13 per GB 
                                                                                                                                   
 OVERALL TOTAL                                                                                                               $0.00 
──────────────────────────────────
4 cloud resources were detected:
∙ 1 was estimated, it includes usage-based costs, see https://infracost.io/usage-file
∙ 3 were free:
  ∙ 1 x azurerm_log_analytics_solution
  ∙ 1 x azurerm_resource_group
  ∙ 1 x azurerm_sentinel_data_connector_microsoft_cloud_app_security